sirepo/uri_router.py

```diff
@@ -157,11 +167,14 @@ def register_api_module(module=None):
     if hasattr(m, 'init_apis'):
         m.init_apis()
     # It's ok if there are no APIs
-    for n, o in inspect.getmembers(m):
-        if n.startswith(_FUNC_PREFIX) and inspect.isfunction(o):
+    if not hasattr(m, 'API'):
+        return
+    c = m.API
```

Comment:
change to `m.Request`.

Reply:
3ccfbc6e956420f7da6fc23176cdebb7ae57cfc9